### ISSUE TITLE
signal 11 (SIGSEGV) Cause: null pointer dereference

### ISSUE THREAD

**Issue by ronitvinocha** [flutter/flutter#145391]
@flutter-symbolizer-bot this

```
03-19 16:46:41.745 24236 24236 F DEBUG   : *** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
03-19 16:46:41.745 24236 24236 F DEBUG   : Build fingerprint: 'realme/RMX3461/RE54BFL1:13/TP1A.220905.001/R.15e7d74_1-2764f:user/release-keys'
03-19 16:46:41.745 24236 24236 F DEBUG   : Revision: '0'
03-19 16:46:41.745 24236 24236 F DEBUG   : ABI: 'arm64'
03-19 16:46:41.745 24236 24236 F DEBUG   : Timestamp: 2024-03-19 16:46:41.049524833+0530
03-19 16:46:41.745 24236 24236 F DEBUG   : Process uptime: 75s
03-19 16:46:41.745 24236 24236 F DEBUG   : Cmdline: com.bigbasket.mobileapp.debug
03-19 16:46:41.745 24236 24236 F DEBUG   : pid: 23282, tid: 23387, name: 1.ui  >>> com.bigbasket.mobileapp.debug <<<
03-19 16:46:41.745 24236 24236 F DEBUG   : uid: 10542
03-19 16:46:41.745 24236 24236 F DEBUG   : signal 11 (SIGSEGV), code 1 (SEGV_MAPERR), fault addr 0x0000000000000000
03-19 16:46:41.745 24236 24236 F DEBUG   : Cause: null pointer dereference
03-19 16:46:41.745 24236 24236 F DEBUG   :     x0  0000000000000000  x1  b4000077d99adcb0  x2  0000000000000000  x3  0000000061766f4e
03-19 16:46:41.745 24236 24236 F DEBUG   :     x4  0000007748c07970  x5  0000000061766f4e  x6  b40000775627ae58  x7  0000000000000008
03-19 16:46:41.745 24236 24236 F DEBUG   :     x8  0000000000000100  x9  b4000077e00a7008  x10 0000000000000000  x11 0000000000000000
03-19 16:46:41.745 24236 24236 F DEBUG   :     x12 00000078058445f8  x13 000000794090d038  x14 000000000000000d  x15 0000000000000000
03-19 16:46:41.745 24236 24236 F DEBUG   :     x16 0000007817030648  x17 0000007940889ec0  x18 00000077fc8ca000  x19 b4000077de9cf0a0
03-19 16:46:41.745 24236 24236 F DEBUG   :     x20 b40000773bc10300  x21 b4000077de9cf0a0  x22 b4000077d9865500  x23 0000007816ba4a78
03-19 16:46:41.745 24236 24236 F DEBUG   :     x24 b400007756284250  x25 0000007805619000  x26 b4000077f2f6bc00  x27 000000761a42d1a0
03-19 16:46:41.745 24236 24236 F DEBUG   :     x28 0000000800000076  x29 00000078057f5ae0
03-19 16:46:41.745 24236 24236 F DEBUG   :     lr  0000007816ae625c  sp  00000078057f57c0  pc  0000007816ae625c  pst 0000000080001000
03-19 16:46:41.745 24236 24236 F DEBUG   : backtrace:
03-19 16:46:41.745 24236 24236 F DEBUG   :       #00 pc 00000000020a625c  /data/app/~~XP5-hlfyN0E8xDCrf6g7Qw==/com.bigbasket.mobileapp.debug-LeANIESWmdo7n1cmDZJb1A==/lib/arm64/libflutter.so (BuildId: 883b4b008c0a3b943ac32e303b4b7c2af57acca4)
03-19 16:46:41.745 24236 24236 F DEBUG   :       #01 pc 00000000020a5bf8  /data/app/~~XP5-hlfyN0E8xDCrf6g7Qw==/com.bigbasket.mobileapp.debug-LeANIESWmdo7n1cmDZJb1A==/lib/arm64/libflutter.so (BuildId: 883b4b008c0a3b943ac32e303b4b7c2af57acca4)
03-19 16:46:41.745 24236 24236 F DEBUG   :       #02 pc 00000000020a5a44  /data/app/~~XP5-hlfyN0E8xDCrf6g7Qw==/com.bigbasket.mobileapp.debug-LeANIESWmdo7n1cmDZJb1A==/lib/arm64/libflutter.so (BuildId: 883b4b008c0a3b943ac32e303b4b7c2af57acca4)
03-19 16:46:41.745 24236 24236 F DEBUG   :       #03 pc 00000000020a6c74  /data/app/~~XP5-hlfyN0E8xDCrf6g7Qw==/com.bigbasket.mobileapp.debug-LeANIESWmdo7n1cmDZJb1A==/lib/arm64/libflutter.so (BuildId: 883b4b008c0a3b943ac32e303b4b7c2af57acca4)
03-19 16:46:41.745 24236 24236 F DEBUG   :       #04 pc 0000000000012238  [anon:dart-code]
```

**Comment by ronitvinocha** [comment]
@flutter-symbolizer-bot  

[!] Flutter (Channel master, 3.21.0-10.0.pre.51, on macOS 14.2.1 23C71 darwin-arm64, locale en-IN)
    ! Warning: `dart` on your path resolves to /opt/homebrew/Cellar/dart/3.0.5/libexec/bin/dart, which is not inside your current Flutter SDK checkout at /Users/ronitvinocha/development/flutter. Consider
      adding /Users/ronitvinocha/development/flutter/bin to the front of your path.
[✓] Android toolchain - develop for Android devices (Android SDK version 34.0.0)
[✗] Xcode - develop for iOS and macOS
    ✗ Xcode installation is incomplete; a full installation is necessary for iOS and macOS development.
      Download at: https://developer.apple.com/xcode/
      Or install Xcode via the App Store.
      Once installed, run:
        sudo xcode-select --switch /Applications/Xcode.app/Contents/Developer
        sudo xcodebuild -runFirstLaunch
    ✗ CocoaPods not installed.
        CocoaPods is used to retrieve the iOS and macOS platform side's plugin code that responds to your plugin usage on the Dart side.
        Without CocoaPods, plugins will not work on iOS or macOS.
        For more info, see https://flutter.dev/platform-plugins
      To install see https://guides.cocoapods.org/using/getting-started.html#installation for instructions.
[✓] Chrome - develop for the web
[✓] Android Studio (version 2023.1)
[✓] Connected device (2 available)
[✓] Network resources

! Doctor found issues in 2 categories.

**Comment by huycozy** [comment]
Hi @ronitvinocha,
When opening an issue, please try to provide all information indicated in the template so that we can address the issue properly (See bug issue template [here](https://github.com/flutter/flutter/issues/new?assignees=&labels=&projects=&template=2_bug.yml))

This looks like a crash in production, so please provide the following information:
- The list of affected device info (OS, model) and the percentage occurrence of the crash
- The entire crash stack trace
